src/mud/examples/adcirc.py

```diff
@@ -171,22 +171,26 @@ def adcirc_ts_plot(
     ax.set_ylabel("Water Elevation (m)")
     ax.set_xlabel("Time")
     _ = ax.set_title("")
-
-    save_figure(
-        "adcirc_full_ts", save_path, close_fig=close_fig, dpi=dpi, bbox_inches="tight"
-    )
+    if save_path:
```

Comment:
Should be 
`if save_path is not None:`
See [PEP 8](https://peps.python.org/pep-0008/)